### PR TITLE
Fix/theme and head section

### PR DESCRIPTION
**Pull request by VitaliyAndrushko**

### CODE REVIEW

File: src/pages/main/HeadSection.js [hunk 1]
```diff
@@ -61,10 +65,9 @@ export const HeadSection = props => (
         justifyContent='center'
         width={[200, 300, 350]}
         href='https://starylev.com.ua/'
-        color={themeGet('colors.white', 'white')(props)}
+        color={themeGet('colors.mainWhite', 'white')(props)}
```
Comment (sancraz) [inline src/pages/main/HeadSection.js:68]: why did you change "white" to "mainWhite"? let`s use "white" for #fff as it is. And other colors which are close to white with different name

**Review by sancraz — APPROVED** [review]

File: src/pages/main/HeadSection.js (hunk 1 above)
Comment (VitaliyAndrushko) [reply src/pages/main/HeadSection.js:68]: We have white as '#fff' and mainWhite witch we use as main white
